gecko_client/pchain_types.go

```diff
@@ -1,14 +1,14 @@
 package gecko_client
 
 // ============= Blockchain ====================
-type BlockchainCreationInfo struct {
+type UnsignedTransaction struct {
 	UnsignedTx string 	`json:"unsignedTx"`
 }
 
-type CreateBlockchainResponse struct {
-	JsonRpcVersion string	`json:"jsonrpc"`
-	Result BlockchainCreationInfo	`json:"result"`
-	Id int	`json:"id"`
+type CreateUnsignedTransactionResponse struct {
```

Comment:
I'd like to keep the mapping of 1 response object = one API call, because they could add more fields to their response objects at any time. Rather than changing this `CreateBlockchainResponse` object, can we create a new `CreateUnsignedTransactionResponse` object that returns a type of `UnsignedTransactionInfo`?

Reply:
done